terraform/account/region/waf.tf

```diff
@@ -400,3 +400,22 @@ fields @message
 | limit 100
 QUERY
 }
+
+resource "aws_cloudwatch_query_definition" "ac4_blocked_logged_in_uris" {
+  name            = "WAF - Blocked Logged-In URI Access"
```

Comment:
Could you make WAF - WAF/ 

When you do the slashes it puts it in the nice folders in aws :)